tools/sandbox-bootstrap.py

```
@@ -1518,6 +1518,17 @@ def cmd_validate(args: argparse.Namespace) -> int:
                         print(f"  {skip_id:<3} {skip_purpose:<18} {_format_outcome('skipped')}  (prior tier failed)")
                 break
 
+    recipe_errors: list[str] = []
+    if SANDBOX_RECIPE_PATH.exists():
+        try:
+            from sandbox.recipe import load_recipe, validate_recipe
+            recipe = load_recipe(SANDBOX_RECIPE_PATH)
+            recipe_errors = validate_recipe(recipe, root=ROOT)
+        except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)